etl/chart_revision_suggester.py

```diff
@@ -0,0 +1,567 @@
+# type: ignore
```

Comment:
Might be useful to reference the original file and say it was copied from there.